common/djangoapps/edraak_ratelimit/models.py

```diff
@@ -37,3 +37,5 @@ class StudentAccountLock(LoginFailures):
 
     class Meta:
         proxy = True
+        managed = False
```

Comment:
Some documentation is required here, we write together something meaningful here.